frontend/src/components/AdvancedSearch.vue

```diff
@@ -208,17 +208,17 @@ const fullScopes = computed((): SearchScope[] => {
       }),
     },
     {
-      id: "review_status",
-      title: t("issue.advanced-search.scope.review-status.title"),
-      description: t("issue.advanced-search.scope.review-status.description"),
+      id: "approval_status",
```

Comment:
Also need to update the `UIIssueFilterScopeIdList` in `frontend/src/utils/v1/issue/ui-filter.ts`

Reply:
Actually we need to change all `review_status` to `approval_status` in the frontend

Reply:
Done. Was thinking of rebranding approval status to issue status later.